src/node/pipeline.spec.ts

```diff
@@ -60,6 +60,52 @@ describe('pipeline', () => {
     assert.strictEqual(await result3, 'hello');
   });
 
+  it('works consistently with streams', async () => {
+    assert.deepStrictEqual(
+      await pipeline([undefined, 1, 2, true, 3, [4, [5], 6, 7]], new PassThrough({ objectMode: true }), toString),
+      '12true34,5,6,7'
+    );
+    await assert.rejects(async () => pipeline([null], new PassThrough({ objectMode: true }), toString), {
+      name: 'TypeError',
+      message: 'May not write null values to stream',
+    });
+    await assert.rejects(async () => pipeline([undefined], new PassThrough({ objectMode: false }), toString), {
+      name: 'TypeError',
+      message: 'The "chunk" argument must be of type string or an instance of Buffer or Uint8Array. Received undefined',
+    });
+    await assert.rejects(async () => pipeline([1], new PassThrough({ objectMode: false }), toString), {
+      name: 'TypeError',
+      message:
+        'The "chunk" argument must be of type string or an instance of Buffer or Uint8Array. Received type number (1)',
+    });
+    await assert.rejects(async () => pipeline([1], new PassThrough({ objectMode: false }), toString), {
```

Comment:
This testcase is repeating the one above it.

Reply:
Cut-n-paste error, fixed.